### ISSUE TITLE
Incorrect Order in Automatic make.

### ISSUE THREAD

**Issue by Christian-B** [SpiNNakerManchester/SupportScripts#14]
Reported by @lplana (Thanks)

automatic_make.sh

In the section sPyNNaker/neural_modelling/
source $PWD/setup
is after the make

Is this correct?

**Comment by alan-stokes** [comment]
ermmm......... given that script was built originally to a make file setup that I think has been changed many times over the years...... it might have been needed at one point to be that way for some reason. 

but if its causing confusion now, and has no harm being switched, I've got absolutely no issues to switch them around. As long as we have a compiler script, that works in the end. 

to add more confusion, if its done afterwards, and the only thing after spynnaker is gfe. which should not have any dependencies on spynnaker at all......... could the spynnaker source setup be removed completely? 

I guess this is a legacy from when spynnaker was spynnaker7 and we had the external devices module that was separate, as that would have dependencies on spynnaker/spynnaker7/spynnaker8.

so unless it causes issues, id remove the line completely. 
anyhow, thats my 2 pence.

**Comment by andrewgait** [comment]
All that script is currently does is set the NEURAL_MODELLING_DIRS variable, and given that the current installation instructions tell the user to do this before going on to manual make or automatic_make, we should probably take it out.

In addition, sourcing the setup file in spinnaker_tools in this automatic_make script is also probably unnecessary for similar reasons - the user has already set the necessary environment variables themselves by this point.

**Comment by alan-stokes** [comment]
it might be that we did'nt ask users to set environment variables when we first stated, and so you had to source setup each time....... OHHH, no, it was coz i refused to install it to begin with (as pycharm is happy running stuff without installs), and so i did not have the environment variables, and so i put the source setup to get them. lol. well im preitty sure ive lost that battle years ago. so remove
